actions/plan/artifact-repositories/download-many.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+export D2L_ARTIFACT_REPOSITORIES_TMP_DIR=$(mktemp -d)
+trap onexit EXIT
+onexit() {
+	set +u
+
+	rm -r "${D2L_ARTIFACT_REPOSITORIES_TMP_DIR}" 2> /dev/null || true
+}
+
+errors=$(echo "${CONFIG}" \
+	| yq -o=json . - \
```

Comment:
Do we want to support YAML here, or does `yq` help for json values too?

Reply:
I was thinking of just supporting it, yeah. I could see not for consistency sake